pkgs/by-name/ad/advanced-rest-client/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, appimageTools, makeDesktopItem, fetchurl }:
+let
+  pname = "advanced-rest-client";
+  desktopItem = makeDesktopItem {
+    name = pname;
+    desktopName = pname;
+    comment = "An API consumption tool for developers";
+    exec = pname;
+  };
+in
+appimageTools.wrapType2 rec {
+  inherit pname;
+  version = "17.0.9";
+
+  src = fetchurl {
+    url = "https://github.com/advanced-rest-client/arc-electron/releases/download/v${version}/arc-linux-${version}-x86_64.AppImage";
+    hash = "sha256-BGHrhv+tfuczlkpSfBuF4GFURU9lVjHXvvXqWxEVd9U=";
+  };
+
+  extraInstallCommands = ''
+    mkdir -p $out/share
+    cp -rt $out/share ${desktopItem}/share/applications
+    mv $out/bin/${pname}-${version} $out/bin/${pname}
```

Comment:
```suggestion
```

No longer needed as of nixos 24.05

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/247651/commits/302ba76deeebc18263fb000293af2a4106c3449f